ts/parsers/graphicFrameParser.ts

```diff
@@ -1,6 +1,10 @@
 //Graphic frame node includes tables, charts and diagrams
 
 import { getAttributeByPath } from "../helpers/attributesHandler";
+import * as isEmpty from "lodash.isempty";
```

Comment:
as discussed

Reply:
wasted a lot of time on this little minor thing. 

It was not importing as it needed to be added as dev depencies.
as @types/loadash.isEmpty in order to import it directly. 

In order to do that interop flag needed to be set. 
![interop](https://user-images.githubusercontent.com/16796013/125193202-4bb5b400-e25c-11eb-9d6e-6e0d342e04d3.png)


Adding that flag in tsconfig breaks ziphanlder and xml2json imports. 
therefore change needed to done in those packages as well to do the exports differently. 

and so many problems one after another so leaving this as is. 

![noregistry](https://user-images.githubusercontent.com/16796013/125193264-9afbe480-e25c-11eb-9e5b-e5b3991f0fdf.png)
![xml2js](https://user-images.githubusercontent.com/16796013/125193265-9b947b00-e25c-11eb-8bed-83bd0e45935a.png)


As it is is importing directly from isEmpty of lodash and isEmpty is the only function that is exported from there, so it doesn't really make a difference, it would have been bad if we imported the whole lodash.

Reply:
@rabi92 not even `import { isEmpty } from "lodash.isempty";` note the {}

Reply:
nope, as isEmpty function is exported directly as module.exports.

![isEmpty](https://user-images.githubusercontent.com/16796013/125194732-95ee6380-e263-11eb-910a-08893e3b406e.png)


But when you use the type Definition. it exports like this and then you can import like you mentioned.

![typeDef](https://user-images.githubusercontent.com/16796013/125194748-a3a3e900-e263-11eb-809a-3b177f761095.png)

Reply:
yes ofc, it's exported directly on module so it's considered the default export. import * is fine.